reference-architecture/vmware-ansible/README.md

```diff
@@ -0,0 +1,103 @@
+# The Reference Architecture OpenShift on VMware
+This repository contains the scripts used to deploy an OpenShift environment based off of the Reference Architecture Guide for OpenShift 3.2 on VMware
+
+## Overview
+The repository contains Ansible playbooks which deploy 3 Masters, 2 infrastructure nodes and 3 application nodes. All nodes could utilize anti-affinity rules to separate them on the number of hypervisors you have allocated for this deployment. The playbooks deploy a Docker registry and scale the router to the number of Infrastruture nodes.
+
+![Architecture](images/OSE-on-VMware-Architecture.jpg)
+
+## Prerequisites
+Internal DNS should be set up to reflect the number of nodes in the environment. The default VM network should have a DHCP server set up for initial provisioning. Lastly, in the var/infrastructure.yaml ip information should also be provided for the nodes in question. Here, VM guest names and other items can be customized.
+
+### OpenShift Playbooks
+The code in this repository handles all of the VMware specific components except for the installation of OpenShift. We rely on the OpenShift playbooks from the openshift-ansible-playbooks rpm. You will need the rpm installed on the workstation before using ose-on-aws.py.
+
+```
+subscription-manager repos --enable rhel-7-server-optional-rpms
+subscription-manager repos --enable rhel-7-server-ose-3.2-rpms
+rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm*
+yum -y install atomic-openshift-utils \
+                 python2-boto \
+                 git \
+                 python-netaddr \
+                 python-httplib2
+
+git clone https://github.com/dannvix/pySphere && cd pySphere/ && python setup.py install
+git clone https://github.com/vmware/pyvmomi && cd pyvmomi/ && python setup.py install
```

Comment:
Not necessary for this iteration, but should probably work with @tdawson on getting these deps packaged up properly.